_data/blogs.yml

```diff
@@ -42,12 +42,6 @@
   article: Exploring Google Code In
   article_link: http://abhijeetmanohar.blogspot.in/2016/12/hi-friends-long-time-of-splashing-codes.html
 
-- name: Adarsh Kumar
-  blog: https://abhijeetmanohar.blogspot.com
-  blog_img: adarsh-blog2.png
-  article: What I learned after participating in GCI
-  article_link: https://idealadarsh.wordpress.com/2016/12/08/what-i-learned-after-participating-in-google-code-in/
-
```

Comment:
Check this https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/_data/blogs.yml#L23
and this https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/_data/blogs.yml#L46
and this https://github.com/fossasia/gci16.fossasia.org/blob/gh-pages/_data/blogs.yml#L52

Adarsh 2 is like made by mixing the first 2 Adarsh Kumar entry and Abhijeet Manohar Entry